src/app/Instance/ErrorHandling/ErrorHandlingDetail.tsx

```diff
@@ -21,6 +31,8 @@ export interface ErrorHandlingDetailProps {
   schema?: object;
   isSchemaLoading: boolean;
   apiError?: string;
+  onEdit: () => void;
+  isEditDisabled: boolean;
```

Comment:
`ErrorHandlingDetail` now renders the "Edit" button, previously located inside `ErrorHandlingTabContent`.
Adding some props to plug it to the parent component.